packages/httpsnippet-client-api/test/index.test.js

```diff
@@ -1,42 +1,64 @@
 /* eslint-disable global-require */
 /* eslint-disable import/no-dynamic-require */
 /* eslint-disable mocha/no-setup-in-describe */
+const chai = require('chai');
```

Comment:
If you're curious why I'm using [chai](https://npm.im/chai) and [mocha](https://mochajs.org/) within this repo instead of Jest it's because I've been slowly moving our open source projects that have code that can be run in a browser to have tests using Chai + Mocha as you can run the same tests in a real browser with [Karma](https://karma-runner.github.io/latest/index.html) with **extremely** minimal effort.

Projects that have been migrated over (so far) are [fetch-har](https://github.com/readmeio/fetch-har) and [openapi-parser](https://github.com/readmeio/openapi-parser).

Reply:
people need to chill out with their pseudo-ethnic package naming conventions lmao